fiftyone/operators/executor.py

```diff
@@ -468,19 +468,23 @@ def dataset_id(self):
     @property
     def view(self):
         """The :class:`fiftyone.core.view.DatasetView` being operated on."""
-        if self._view is not None:
+        if self._view:
```

Comment:
It's critical that this remain unchanged. `bool(DatasetView)` checks if `len(view) > 0`, which is not the same thing as `view is not None`.